backend/entities/Models/PropertyFilter.cs

```diff
@@ -191,6 +198,7 @@ public override bool IsValid()
                 || this.SWLongitude.HasValue
                 || !String.IsNullOrWhiteSpace(this.ProjectNumber)
                 || this.IgnorePropertiesInProjects == true
```

Comment:
Both these boolean checks are probably not ideal.  I think it should be just checking if they not null `.HasValue`.  But it isn't critical for our use case.